mirror/src/main/java/com/bakdata/quick/mirror/MirrorController.java

```diff
@@ -73,4 +73,14 @@ public Single<HttpResponse<MirrorValue<List<V>>>> getAll() {
         log.debug("Request for all existing keys.");
         return this.queryService.getAll();
     }
+
+    /**
+     * Fetches a range of values over a given key and a range field.
+     */
+    @Get("range/{key}")
```

Comment:
Just for consistency:
```suggestion
    @Get("/range/{key}")
```